cmd/escrow/config.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"github.com/docker/notary"
+	"github.com/docker/notary/storage"
+	"github.com/docker/notary/trustmanager"
+	"github.com/docker/notary/trustmanager/remoteks"
+	"github.com/docker/notary/utils"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"net"
```

Comment:
This is how gogland is arranging them for me :-/ I need to try breaking them up into our normal format and see what it does with new imports that get auto-added. If we're all having to constantly tweak the import list though, maybe we stop trying so hard to organize it. It looks nicer the way we've been doing it, but I think everyone also loves just having imports auto-managed.